checker/build.gradle

```diff
@@ -17,6 +17,7 @@ dependencies {
     // File Utilities takes priority over this version, in the fat jar
     // file. :-( So update it and re-build it locally when updating this.
     implementation 'org.plumelib:reflection-util:0.2.2'
+    implementation 'org.plumelib:plume-util:1.1.5'
```

Comment:
Just to be clear, do NOT add this to checker-framework/docs/developer/release/maven-artifacts/checker-pom.xml.  That is the pom file for the fat jar that includes all dependencies.